### PR TITLE
PDClient: Add function to call pause checker API

### PR DESCRIPTION
**Pull request by peng1999**
Signed-off-by: Peng Guanwen <pg999w@outlook.com>

### What problem does this PR solve? 

https://github.com/tikv/client-java/issues/268 needs to call pause checker in Java client. This PR adds the function.

### What is changed and how it works?
Added functions:

- `PDClient.keepPauseChecker`
- `PDClient.stopKeepPauseChecker`
- `PDClient.resumeChecker`

### Check List 

Tests 

 - Integration test

Code changes

 - Has exported function/method change

Related changes

 - Need to be included in the release note

### CODE REVIEW

File: src/test/java/org/tikv/common/PDClientTest.java [hunk 1]
```diff
@@ -68,6 +68,15 @@ public void testTso() throws Exception {
     }
   }
 
+  @Test
+  public void testPauseCheck() throws Exception {
+    try (PDClient client = session.getPDClient()) {
+      client.keepPauseChecker(PDChecker.Merge);
+      Thread.sleep(6000);
```
Comment (marsishandsome) [inline src/test/java/org/tikv/common/PDClientTest.java:75]: can we get the current status (pause or not)  and add assert here?

Comment (peng1999) [reply src/test/java/org/tikv/common/PDClientTest.java]: Ok. `isCheckerPaused` is added.

File: src/test/java/org/tikv/common/PDClientTest.java [hunk 2]
```diff
@@ -68,6 +68,22 @@ public void testTso() throws Exception {
...
+      PDChecker checker = PDChecker.Merge;
+      client.keepPauseChecker(checker);
+      Thread.sleep(1000);
+      assertTrue(client.isCheckerPaused(checker));
+
+      client.stopKeepPauseChecker(checker);
+      Thread.sleep(1000);
+
+      client.resumeChecker(checker);
+      assertFalse(client.isCheckerPaused(checker));
```
Comment (marsishandsome) [inline src/test/java/org/tikv/common/PDClientTest.java:83]:
```suggestion
    try (PDClient client = session.getPDClient()) {
      PDChecker checker = PDChecker.Merge;
      for(int i = 0; i < 2; i ++) {
        client.keepPauseChecker(checker);
        Thread.sleep(1000);
        assertTrue(client.isCheckerPaused(checker));
  
        client.stopKeepPauseChecker(checker);
        Thread.sleep(1000);
  
        client.resumeChecker(checker);
        assertFalse(client.isCheckerPaused(checker));
      }
```

**Review by marsishandsome — COMMENTED** [review]
let's merge this PR after https://github.com/tikv/pd/pull/4092 is merged

**Comment by marsishandsome** [comment]
run `mvn clean package -DskipTests` to format the code

**Review by marsishandsome — APPROVED** [review]
LGTM

**Comment by mzygQAQ** [comment]
pause checker is not enough for bulkload, we should also remove pd's schedulers

**Pull request merged**